src/ukify/ukify.py

```diff
@@ -583,16 +595,16 @@ def call_systemd_measure(uki, opts, profile_start=0):
                 extra += [f'--public-key={pub_key}']
             extra += [f'--phase={phase_path}' for phase_path in group or ()]
 
-            print('+', shell_join(cmd + extra))
-            pcrsig = subprocess.check_output(cmd + extra, text=True)
+            print('+', shell_join(cmd + extra))  # type: ignore
+            pcrsig = subprocess.check_output(cmd + extra, text=True)  # type: ignore
             pcrsig = json.loads(pcrsig)
             pcrsigs += [pcrsig]
 
         combined = combine_signatures(pcrsigs)
         uki.add_section(Section.create('.pcrsig', combined))
 
 
-def join_initrds(initrds):
+def join_initrds(initrds: list[Path]) -> Union[Path, bytes, None]:
```

Comment:
`[bytes, Path, None]` ?

Reply:
I'm not sure that's valid for a union, though mypy doesn't complain. The nice syntax for this would be `Path | bytes | None`, but we're prevented from using that by Python 3.9 backwards compat.